db/app/models/reviews.m.js

```diff
@@ -86,26 +81,88 @@ exports.updateReviewById = (id, votes) => {
   }
 };
 
-exports.selectReviews = () => {
+exports.selectReviews = async ({
+  sort_by = "created_at",
+  order = "DESC",
+  category = "",
+  ...query
+}) => {
+  let correctQueries = new Set(["sort_by", "order", "category"]);
+
+  for (let key in query) {
+    if (!correctQueries.has(key) && key.length > 0) {
+      return Promise.reject({
+        status: 400,
+        message: "Bad request, incorrect method",
+      });
+    }
+  }
+
+  let whereStr = "";
+  let sortByOptions = new Set([
+    "title",
+    "designer",
+    "owner",
+    "review_img_url",
+    "review_body",
+    "category",
+    "created_at",
+    "votes",
+  ]);
+  let checkSortOptions = sortByOptions.has(sort_by);
+
+  if (checkSortOptions === false) {
+    return Promise.reject({
+      status: 400,
+      message: "Bad request, incorrect sort_by",
+    });
+  } else {
+    searchSort_by = sort_by;
```

Comment:
this variable looks like its not being used anymore - is it left over from before the refactor?